src/event/trigger.js

```diff
@@ -136,7 +139,17 @@ jQuery.extend( jQuery.event, {
 
 					// Prevent re-triggering of the same event, since we already bubbled it above
 					jQuery.event.triggered = type;
+
+					if ( event.isPropagationStopped() ) {
+						lastElement.addEventListener( type, stopPropagationCallback );
```

Comment:
I think it would be possible to get here with `lastElement === undefined` if the incoming `event` already had propagation stopped? Maybe initialize with `lastElement = elem`?